appveyor.yml

```diff
@@ -26,13 +33,13 @@ build_script:
   - python -W ignore setup.py -q bdist_wheel
 
 test_script:
-  - python setup.py test -q
+  - python setup.py -q test
 
 artifacts:
   - path: 'dist\*.whl'
     name: wheel
 
 deploy_script:
-  - ps: if ($env:APPVEYOR_REPO_TAG -eq $TRUE) { pip install twine; twine upload dist/* }
+  - ps: if ($env:APPVEYOR_REPO_TAG -eq $TRUE) { pip install twine; twine --skip-existing upload dist/* }
```

Comment:
This part was not so good: https://ci.appveyor.com/project/mgedmin/zope-interface/builds/28777006/job/g40lp9dhkof95mf2

Fixed in 1a84324.